.github/workflows/DailyBuild.yaml

```
@@ -0,0 +1,48 @@
+name: Daily Aplication Build
+
+# Builds project every day at 5:00
+on:
+  schedule:
+    - cron: "0 5 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    # https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python
+    steps:
+      # checkout the repository
+      - uses: actions/checkout@v2
+        with:
+          ref: develop #checkout specifically develop
+
+      - name: prepare python pre-built environnement
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: test python is functionnal
+        run: python -c "import sys; print(sys.version)"
+      
+      - name: Pepare dependencies python, cmake
+        run: |
+          python -m pip install --upgrade pip
+          sudo apt install cmake -y
+
+      - name: init submodule
+        run: git submodule update --init
+        
+      # Init the virtual environnement (project script)
+      - name: Prepare venv
+        run: ./scripts-ci/init-venv.sh
+
+      # Build the project (project script)
+      - name: Build the project
+        run: ./scripts-ci/build.sh
+
+      # Upload the artifact
+      - name: Upload the application binaries
+        uses: actions/upload-artifact@v2
+        with:
+          name: Application binaries
+          path: fprime/Ref/build-*
```

I think it's the wrong path.
```suggestion
          path: App/build-*
```